### ISSUE TITLE
Auto complete address dropdown appears behind the Text Input.

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#14312]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to Workspace => Connect Bank Account => Connect Manually => Enter valid information for first 2 steps.
2. At step 3, tap on Personal Address, type some text to make the auto complete address dropdown appears, try to scroll up.
3. Notice that the dropdown appears behind the TextInput

## Expected Result:
The dropdown shouldn’t appear behind the TextInput

## Actual Result:
the dropdown appears behind the TextInput

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?

- [ ] iOS / native

**Version Number:** v1.2.54-2
**Reproducible in staging?:** y 
**Reproducible in production?:** Can't check
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:**  

https://user-images.githubusercontent.com/43996225/212491927-dae91deb-1d41-4811-bba9-bd66e42014a7.MP4

https://user-images.githubusercontent.com/43996225/212491934-95936ea9-6e4b-4e32-bbcf-2a49051df42f.MP4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @hungvu193 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1673688358786309

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kadiealexander** [comment]
Chatted to @hungvu193 and updated the OP to reflect the actual bug.

### Bug0 Triage Checklist
_Note: see [this SO](https://stackoverflow.com/c/expensify/questions/14418) for more information._
- [x] The "bug" is actually a bug 
- [x] The bug is not a duplicate report of an existing GH.
  - If it is, close the GH and add any novel details to the original GH instead 
- [x] The bug is reproducible, following the reproduction steps.
  - [x] If the GH doesn’t have steps to reliably reproduce the bug and you figure it out, then please add them.
  - [x] If you’re unable to reproduce the bug, add the `Needs reproduction` label. 
  - [x] Comment on the issue outlining the steps you took to try to reproduce the bug, your results and tag the issue reporter and the Applause QA member who created the issue. Ask them to clarify reproduction steps and/or to check the reproduction steps again. Close issue. 
- [x] The GH template is filled out as fully as possible
  - The GH body and title are clear (ie. could an external contributor understand it and work on it?) 
- [x] The GH was created by an Expensify employee or a QA tester 
- [x] If the bug is an OldDot issue, you should decide whether it is widespread enough to justify fixing or it is better to wait for reunification. If it’s better to wait, close the GH & provide this justification 
- [x] If there's a link to Slack, check the discussion to see if we decided not to fix it 
- [x] Decide if the GH should be resolved by an External contributor or Internal engineer, add the appropriate label

https://user-images.githubusercontent.com/59587260/213292761-57ad4c58-c4bb-43ac-ade4-32414a2bc13f.MOV

**Comment by Karim-30** [comment]
### Proposal

Remove the `overflowAuto` style.

```diff
+++ b/src/components/AddressSearch.js
@@ -215,7 +215,6 @@ const AddressSearch = (props) => {
                             displayListViewBorder && styles.borderTopRounded,
                             displayListViewBorder && styles.borderBottomRounded,
                             displayListViewBorder && styles.mt1,
-                            styles.overflowAuto,
                             styles.borderLeft,
                             styles.borderRight,
                         ],
```


https://user-images.githubusercontent.com/51829206/212557846-795a0d9c-133f-4392-b65f-e3e5cf09cc26.mov

**Comment by bernhardoj** [comment]
### **Proposal**

The GooglePlacesAutocomplete component uses a VirtualizedList internally even though it always returns max. of 5 items. To prevent the scrolling, we can pass `disableScroll` props to the component.

```diff
diff --git a/src/components/AddressSearch.js b/src/components/AddressSearch.js
index 4d465c990a..77aeb76c76 100644
--- a/src/components/AddressSearch.js
+++ b/src/components/AddressSearch.js
@@ -152,6 +152,7 @@ const AddressSearch = (props) => {
         >
             <View style={styles.w100}>
                 <GooglePlacesAutocomplete
+                    disableScroll
                     fetchDetails
                     suppressDefaultStyles
                     enablePoweredByContainer={false}

```

**Result**

https://user-images.githubusercontent.com/50919443/213361060-2b2a8e3c-582e-4515-9fc0-ce99189be7b6.mov

**Comment by s77rt** [comment]
I don't see why the expected behaviour is to disable scroll on the list. I think we should keep the list scroll able and go with @dhairyasenjaliya [proposal's](https://github.com/Expensify/App/issues/14312#issuecomment-1396275440)

**Comment by priyeshshah11** [comment]
I agree with @bernhardoj's proposal as there is no reason to have that list scrollable as it always displays a limited number of items which are all visible on the screen already & the parent view already has a scroll & also to keep it consistent with other platforms.

**Comment by kadiealexander** [comment]
@aimane-chnaif @johnmlee101 any thoughts on the proposals above?

**Comment by johnmlee101** [comment]
👍 sounds good to me.

**Comment by bernhardoj** [comment]
PR is ready for review.

**Comment by johnmlee101** [comment]
Haven't gotten to the follow-up checklist yet!

**Comment by kadiealexander** [comment]
Sorry for the delay guys! I have sent contracts to @aimane-chnaif and @bernhardoj and will pay the bounty + speed bonus once you've accepted. 

@hungvu193 could you please apply in Upwork so I can get your reporting bonus paid?

**Comment by hungvu193** [comment]
Thanks @kadiealexander, just applied!

**Comment by bernhardoj** [comment]
Thank you @kadiealexander!

**Comment by kadiealexander** [comment]
Paid you both! Thanks for the prompt action on that.

**Comment by kadiealexander** [comment]
Everyone is paid! @aimane-chnaif and @johnmlee101 don't forget about [this checklist](https://github.com/Expensify/App/issues/14312#issuecomment-1407029378) please!

**Comment by kadiealexander** [comment]
@johnmlee101 are you happy to close this one out without adding a regression test?

**Comment by johnmlee101** [comment]
Yes! I think we're good to close this out
